modules/core/src/main/java/io/fluo/webindex/core/models/DomainStats.java

```diff
@@ -16,12 +16,9 @@
 
 public class DomainStats {
 
-  private String domain;
```

Comment:
it would probably be better here to add a get method for domain rather than remove it from the class.